numba/tests/test_array_manipulation.py

```diff
@@ -865,6 +870,70 @@ def test_broadcast_to_indexing(self):
             got = cfunc(input_array, shape, idx)
             self.assertPreciseEqual(got, expected)
 
+    if numpy_version >= (1, 20):
```

Comment:
Suggest using `unittest.skipIf` or similar on the test cases opposed to guarding them?